src/backend/twitch-api/resource/auth.ts

```diff
@@ -27,7 +27,7 @@ export class TwitchAuthApi {
                     break;
             }
 
-            // This shouldn only happen if an account is not logged in
+            // This shouldnt only happen if an account is not logged in
```

Comment:
Not sure, but to me, it seems like this should be either `shouldn't`, `should not` or `should`. If I understand the code correctly, I think it should be `should`.

Reply:
do you see any others?

Reply:
i stopped doing comments and started going after what is in the UI so there are lots more that are just comments.